README.md

```diff
@@ -125,6 +125,21 @@ describe('GET /users', function() {
 });
 ```
 
+Or async/await syntax:
+
+```js
+describe('GET /users', function() {
+  it('responds with json', async function() {
+    const response = request(app)
```

Comment:
Did you forget about `await` ?

```javascript
const response = await request(app)
```

Reply:
Oh right, I did. Added a commit with the fix.